web/packages/core/src/shadow-template.ts

```diff
@@ -227,3 +227,37 @@ ruffleShadowTemplate.innerHTML = `
 
     <ul id="context-menu"></ul>
 `;
+
+export const preloaderTemplate = document.createElement("template");
+preloaderTemplate.innerHTML = `
+    <div id="preloader">
+        <style>
+            #preloader {
+                display: flex;
+                flex-direction: column;
+                background: var(--ruffle-blue);
+                width: 100%;
+                height: 100%;
+                padding: 10px auto;
+                align-items: center;
+                justify-content: center;
+            }
+
+            .loadbar {
+                width: 100%;
+                max-width: 316px;
+                height: 10px;
+                margin-top: -15px;
+                background: #253559;
+            }
+
+            .loadbarInner {
+                width: 0px;
+                height: 10px;
+                background: var(--ruffle-orange);
+            }
+        </style>
```

Comment:
.logo {
            	display: var(--logo-display, block);
            }
        </style>
That way, people who don't want to use the Ruffle logo can add this to the main site's CSS:

```
:root {
    --logo-display: none;
}
```